webapi/src/main/scala/org/knora/webapi/responders/v2/OntologyResponderV2.scala

```diff
@@ -3148,12 +3158,18 @@ class OntologyResponderV2 extends Responder {
             throw InconsistentTriplestoreDataException(s"Property $propertyIri contains salsah-gui:guiOrder")
         }
 
-        PropertyInfoContentV2(
+        val propertyDef = PropertyInfoContentV2(
             propertyIri = propertyIri,
             subPropertyOf = subPropertyOf,
             predicates = otherPreds,
             ontologySchema = propertyIri.getOntologySchema.get
         )
+
+        if (!propertyIri.isKnoraBuiltInDefinitionIri && propertyDef.getRdfTypes.contains(OntologyConstants.Owl.TransitiveProperty.toSmartIri)) {
```

Comment:
Does this make sure that you cannot use `owl:transitiveProperty` in a project ontology because of the reasons outlined in FAQ?

Reply:
Yes.